ci-operator/step-registry/redhat-chaos/pod-scenarios-cr/redhat-chaos-pod-scenarios-cr-chain.yaml

```diff
@@ -0,0 +1,12 @@
+
+chain:
+  as: redhat-chaos-pod-scenarios-cr
+  steps:
+  - chain: redhat-chaos-krkn-hub-etcd-tests
```

Comment:
was thinking about this chain, can you see what steps are different between this one and [redhat-chaos-krkn-hub-tests-chain](https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-chaos/krkn-hub-tests/redhat-chaos-krkn-hub-tests-chain.yaml), maybe we can remove the aws in each versioned config and replace it with this test